server/core/src/main/scala/org.apache.daffodil.debugger.dap/Parse.scala

```diff
@@ -364,7 +362,12 @@ object Parse {
         startElement.name.map(_.value).getOrElse("???"),
         /* If sourceReference > 0 the contents of the source must be retrieved through
          * the SourceRequest (even if a path is specified). */
-        new Types.Source(schemaPath.toString, 0),
+        Try(Paths.get(URI.create(startElement.schemaLocation.uriString)).toString())
+          .fold(
+            _ =>
+              new Types.Source(startElement.schemaLocation.uriString, null, 0), // there is no valid path if the location is a schema contained in a jar file; see #76.
```

Comment:
The check passed: https://github.com/apache/daffodil-vscode/runs/5491197876?check_suite_focus=true. Maybe a local formatter difference?

Reply:
I agree to both side that this line does seem long but does not show up in the scala formatting. Does scalafmt not count comment string in the line width? Maybe can the comment can be moved above`new Types.Source(startElement.schemaLocation.uriString, null, 0),` instead of on the same line?